### PR TITLE
Added support for Sina Weibo

### PR DESCRIPTION
**Pull request by davidruisinger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,18 +19,46 @@
...
     "passport-google-oauth": "^0.1.5",
+    "passport-local": "^1.0.0",
     "passport-twitter": "^1.0.2",
-    "debug": "^0.7.4",
-    "passport-github": "^0.1.5",
-    "passport-dribbble": "^1.0.1"
-  }
+    "passport-weibo": "^0.1.2"
+  },
+  "gitHead": "209402a9ff55f097b34db46645373f3d23a0e373",
```
Comment (Maddis1337) [inline package.json]: Please do not check in a generated package.json

**Comment by flavordaaave** [comment]
@Maddis1337 Sorry, I don't really know what to change/remove from the package.json file so I just stripped it down to only contain the basic info including the updated dependencies.

**Comment by Maddis1337** [comment]
Just use the package.json from this repository and add your dependency.

**Comment by flavordaaave** [comment]
@Maddis1337 THX. Done.

File: Readme.md [hunk 2]
```diff
@@ -74,14 +76,14 @@ Note: You may supply the baseURL (your website's root) via the environment varia
...
 
-Point your users to `/auth/{twitter,facebook,github,google}` to have them login (or signup) via the specified module.
+Point your users to `/auth/{twitter,facebook,github,google,dribble,weibo}` to have them login (or signup) via the specified module.
 After that, Auth-Passport completely takes over and redirects the users according to the OAuth(2) flow.
 
 Also You can use `/auth/login` to login on local user collection but it has to be POST method.
 
 ### Usage in Mobile Apps
 
-Auth-Passport was built with usage in mobile Apps in mind. From inside your mobile app, open a browser and point the user to your website's `/auth/{login,twitter,facebook,github}` endpoint. From there, Auth-Passport will take over and guide (i.e. redirect) your user through the different steps needed for each provider, until the user has authorized your app and logged in successfully.
```
Comment (Maddis1337) [inline Readme.md]: could you please readd the local login part? /auth/login
I just noticed, that it also misses in the local login in the usage. Would be great if you could include login there too

File: Readme.md [hunk 3]
```diff
@@ -74,14 +76,14 @@ Note: You may supply the baseURL (your website's root) via the environment varia
 
 ### Usage
 
-Point your users to `/auth/{twitter,facebook,github,google}` to have them login (or signup) via the specified module.
```
Comment (Maddis1337) [inline Readme.md]: please add /auth/login

File: Readme.md (hunk 2 above)
Comment (flavordaaave) [inline Readme.md]: Done.

File: Readme.md (hunk 3 above)
Comment (flavordaaave) [inline Readme.md]: Done.

**Comment by 0ff** [comment]
Published as 0.3.6 @ npmjs

**Pull request merged**